src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
```

Comment:
You don't need it in this task, but maybe better to use enum instead of just types.

Reply:
introduce enum